R_scripts/licordat.R

```diff
@@ -57,27 +60,46 @@ apple <- read_dir("/Users/penn529/Desktop/apple_data/")
 x <- read_dir("/Users/penn529/Desktop/PREMIS/licor_test_data/")
 read_dir("/Users/penn529/Desktop/PREMIS/licor_test_data/SJ_6_1_16/")
 plant <- read_dir("/Users/penn529/Desktop/plant_data/")
+licorDat <- read_dir("/Users/penn529/Documents/GitHub/PREMIS-ghg/licor_data/")
 
 # Create practice df with multiple collars
 # merge with cores_collars.csv
 # practice plotting with ggplot
 d <- data.frame(Timestamp = rep(1:5, times = 12), Flux = runif(120), Collar = rep(1:120, each = 5))
-collardata <- read.csv("../design/cores_collars.csv")
-y <- merge(d, collardata)
+collarDat <- read.csv("/Users/penn529/Desktop/PREMIS/cores_collars.csv")
```

Comment:
We don't want this explicit path, because then (for example) I can't run it on my computer. Use `"../design/cores_collars.csv"` or something like that please.